packages/core/src/components/ModalNew/Modal/Modal.module.scss

```diff
@@ -0,0 +1,12 @@
+.overlay {
+  position: fixed;
+  inset: 0;
+  background-color: rgba(41, 47, 76, 0.7);
```

Comment:
Is it ok? I think we recently changed it to be more adjustable to the themes, no?

Reply:
I'll create a separate task for it